src/convertToObject.js

```diff
@@ -1,12 +1,23 @@
 'use strict';
 
-/**
+/*
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleUpgrated = sourceString
+    .split(';')
+    .map((el) => el.split(':'))
```

Comment:
do u need this `map` method? u can do this on each iteration in the middle of the `reduce` method